src/scribe_data/cli/get.py

```diff
@@ -159,7 +159,7 @@ def prompt_user_download_all():
         else:
             print("Updating all languages and data types...")
             rprint(
-                "[bold red]Note that the download all functionality must use Wikidata dumps to observe responsible Wikidata Query Service usage practices.[/bold red]"
+                "[bold red]Note that the download all functionality must use Wikidata lexeme dumps to observe responsible Wikidata Query Service usage practices.[/bold red]"
```

Comment:
Decided to be explicit here with the dumps as there are Wikidata dumps that are all of Wikidata, but we're using the lexeme subset :) Again just a note on the change! 😊